sw/device/tests/aes_test.c

```diff
@@ -41,11 +41,10 @@ bool test_main(void) {
       .mode = kAesEcb, .key_len = kAes256, .manual_operation = false,
   };
 
-  aes_key_put(key_32_1, aes_cfg.key_len);
-
   // Encode
   aes_cfg.operation = kAesEnc;
   aes_init(aes_cfg);
+  aes_key_put(key_32_1, aes_cfg.key_len);
```

Comment:
nit: any particular reason why these variables have a `_1` suffix?

Reply:
It's kind of a leftover. The referenced document has two examples and this is from example one. Please note that this test application is reworked anyway in PR #2933.